api/src/main/java/edu/cornell/mannlib/vitro/webapp/dao/filtering/IndividualFiltering.java

```diff
@@ -565,5 +565,9 @@ public boolean hasThumb() {
 	public void resolveAsFauxPropertyStatements(List<ObjectPropertyStatement> list) {
 		_innerIndividual.resolveAsFauxPropertyStatements(list);
 	}
+	
+    public Individual get_innerIndividual() {
```

Comment:
Adding a method to unwrap the instance is a bit of a red flag. Looking at the rest of the code, I'm not convinced it's actually necessary - and in any case, if it was done, it's worth considering if it should be part of the interface, and implemented in all implementations (with a class that doesn't have a wrapped instance either returning null or itself).